apps/server/src/modules/h5p-editor/service/h5p-editor.service.ts

```diff
@@ -0,0 +1,39 @@
+import { H5PEditor, cacheImplementations } from '@lumieducation/h5p-server';
+
+import { IH5PEditorOptions, ITranslationFunction } from '@lumieducation/h5p-server/build/src/types';
+import { h5pConfig, h5pUrlGenerator } from './config/h5p-service-config';
+import { ContentStorage } from './contentStorage.service';
+import { Translator } from './h5p-translator.service';
+import { LibraryStorage } from './libraryStorage.service';
+import { TemporaryFileStorage } from './temporary-file-storage.service';
+
+export const H5PEditorService = {
```

Comment:
Service zu Provider umbenennen und in ein provider Odner verschieben.

Reply:
Done